Config.cmake/test_fmi3.cmake

```diff
@@ -133,24 +114,22 @@ set_target_properties(
 set(FAIL_NAME_CHECK 0)
 set(PASS_NAME_CHECK 1)
 
-add_catch2_test(fmi3_capi_basic_test             FMI3)
-add_catch2_test(fmi3_import_start_arrays_test    FMI3)
-add_catch2_test(fmi3_import_variable_test        FMI3)
-add_catch2_test(fmi3_import_convenience_test     FMI3)
-add_catch2_test(fmi3_import_type_definitions_test FMI3)
-add_catch2_test(fmi3_import_model_structure_test FMI3)
+add_catch2_test(fmi3_capi_basic_test               FMI3)
+add_catch2_test(fmi3_import_variable_test          FMI3)
+add_catch2_test(fmi3_import_start_arrays_test      FMI3)
+add_catch2_test(fmi3_import_convenience_test       FMI3)
+add_catch2_test(fmi3_import_model_description_test FMI3)
+add_catch2_test(fmi3_import_type_definitions_test  FMI3)
+add_catch2_test(fmi3_import_model_structure_test   FMI3)
+add_catch2_test(fmi3_import_bouncingball_xml_test  FMI3)
+add_catch2_test(fmi3_import_interface_types_test   FMI3)
 
 add_test(ctest_fmi3_xml_parsing_test fmi3_xml_parsing_test ${FMIL_TEST_DIR}/FMI3/parser_test_xmls/)
 
-add_test(ctest_fmi3_import_xml_test_empty fmi3_import_xml_test ${FMU3_DUMMY_FOLDER})
-add_test(ctest_fmi3_import_xml_test_me fmi3_import_xml_test ${TEST_OUTPUT_FOLDER}/${FMU3_DUMMY_ME_MODEL_IDENTIFIER}_me)
-add_test(ctest_fmi3_import_xml_test_cs fmi3_import_xml_test ${TEST_OUTPUT_FOLDER}/${FMU3_DUMMY_CS_MODEL_IDENTIFIER}_cs)
-add_test(ctest_fmi3_import_xml_test_mf fmi3_import_xml_test ${TEST_OUTPUT_FOLDER}/${FMU3_DUMMY_MF_MODEL_IDENTIFIER}_mf)
-set_tests_properties(ctest_fmi3_import_xml_test_mf PROPERTIES WILL_FAIL TRUE)
```

Comment:
All of these are now just:
`add_catch2_test(fmi3_import_bouncingball_xml_test  FMI3)`